charts/newrelic-logging/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart to deploy New Relic Kubernetes Logging as a DaemonSet, supporting both Linux and Windows nodes and containers
 name: newrelic-logging
-version: 1.13.1
+version: 1.13.2
```

Comment:
@woehrl01 it looks like `nri-bundle/Charts.lock` line 25 will need to be updated with this new value. 

I'm seeing a build error:
```

1 chart(s) linted, 0 chart(s) failed
Error: Chart.lock is out of sync with Chart.yaml
Error: Error linting charts: Error building dependencies for chart 'nri-bundle => (version: "5.0.11", path: "charts/nri-bundle")': Error waiting for process: exit status 1
```

Thank you!

Reply:
@bmcfeely  thanks for pointing that out, I just pushed the fix